### ISSUE TITLE
safeApp.authorise() causes uncaught exception in electron

### ISSUE THREAD

**Issue by theWebalyst** [maidsafe/safe_browser#162]
Operation System: Debian 8/Jesse 
Safe Browser Version: 0.5.1

I'm getting an intermittent problem calling safeApp.authorise() which looks like an error in the browser itself. This seems similar to another issue (which I can't find the number for right now, but as the error info is different I'm logging it separately).

The authorise() call seems to go through (because auth request is presented by the browser), but a modal dialog appears over the browser window with the following text:

```
Uncaught Exception:
TypeError: Cannot read property 'webContents' of null
	at EventEmitter.electron.ipcMain.on (/home/mrh/safe-releases/2017-09-07_Test_19_update/safe-browser-v0.5.1-linux-x64/resources/app/background-process.build.js:1821:6)
	at emitTwo (events.js:106:13)
	at EventEmitter.emit (events.js:194:7)
	at WebContents.<anonymous> (/home/mrh/safe-releases/2017-09-07_Test_19_update/safe-browser-v0.5.1-linux-x64/resources/electron.asar/browser/api/web-contents.js:247:13)
	at emitTwo (events.js:106:13)
	at WebContents.emit (events.js:194:7)
```

The code I'm using is pretty simple, but part of a larger codebase with lots of stuff going on. You can visit it on Test 19 at safe://myfd.rsports. Load the page and then click on the blue cube top right of the web page.

**Comment by theWebalyst** [comment]
Possibly the same or related issues:

https://maidsafe.atlassian.net/browse/MAID-2321
https://github.com/maidsafe/beaker-plugin-safe-authenticator/issues/137

**Comment by theWebalyst** [comment]
Fixed in SAFE Browser 0.5.2 😃
